### ISSUE TITLE
Installtion: The 'picard' MultiQC module broke

### ISSUE THREAD

**Issue by parvathisudha** [ewels/MultiQC#1367]
Hi,

I am getting an issue while installing MultiQC via conda.

```
  multiqc : Oops! The 'picard' MultiQC module broke... 
  Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues 
  If possible, please include a log file that triggers the error - the last file found was:
    ./conda-env/lib/python3.9/site-packages/multiqc/modules/picard/__pycache__/ValidateSamFile.cpython-39.pyc
============================================================
Module picard raised an exception: Traceback (most recent call last):
  File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/multiqc.py", line 569, in run
    output = mod()
  File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/picard/picard.py", line 109, in __init__
    n['ValidateSamFile'] = ValidateSamFile.parse_reports(self)
  File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/picard/ValidateSamFile.py", line 90, in parse_reports
    data = _parse_reports_by_type(self)
  File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/picard/ValidateSamFile.py", line 124, in _parse_reports_by_type
    first_line = filehandle.readline().rstrip()
  File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdb in position 9: invalid continuation byte
```

I am trying to plot WGScoverage metrics of Picard using MultiQC. 
Please help me to solve the issue.

Thanks
Parvathi.

**Comment by ewels** [comment]
Hi @parvathisudha,

Could you please provide some more information about exactly what you have done? You say that the problem was with installation, but the error log looks like it came from a MultiQC analysis run. Are you sure that you're not somehow running MultiQC on the installation directory?

Maybe relevant: [docs](https://multiqc.info/docs/#using-multiqc)

Phil

**Comment by parvathisudha** [comment]
Hi,

This happened during the installation step. 
```
conda install -c bioconda -c conda-forge multiqc
multiqc .
```

> [INFO   ]         multiqc : This is MultiQC v1.9
> [INFO   ]         multiqc : Template    : default
> [INFO   ]         multiqc : Searching   : /geode2/home/u070/parkanha/Carbonate/Application/MultiQC
> Searching 16098 files..  [####################################]  100%          
> [ERROR  ]         multiqc : Oops! The 'gatk' MultiQC module broke... 
>   Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues 
>   If possible, please include a log file that triggers the error - the last file found was:
>     ./conda-env/lib/python3.9/site-packages/multiqc/modules/gatk/varianteval.py
> ============================================================
> Module gatk raised an exception: Traceback (most recent call last):
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/multiqc.py", line 569, in run
>     output = mod()
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/gatk/gatk.py", line 42, in __init__
>     n_reports_found += self.parse_gatk_varianteval()
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/gatk/varianteval.py", line 20, in parse_gatk_varianteval
>     parsed_data = parse_single_report(f['f'])
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/gatk/varianteval.py", line 122, in parse_single_report
>     d[headers[i]] = s
> IndexError: list index out of range
> ============================================================
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [WARNING]  MarkDuplicates : Skipping MarkDuplicates sample 'MarkDuplicates.py' as missing essential fields
> [ERROR  ]         multiqc : Oops! The 'picard' MultiQC module broke... 
>   Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues 
>   If possible, please include a log file that triggers the error - the last file found was:
>     ./conda-env/lib/python3.9/site-packages/multiqc/modules/picard/__pycache__/ValidateSamFile.cpython-39.pyc
> ============================================================
> Module picard raised an exception: Traceback (most recent call last):
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/multiqc.py", line 569, in run
>     output = mod()
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/picard/picard.py", line 109, in __init__
>     n['ValidateSamFile'] = ValidateSamFile.parse_reports(self)
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/picard/ValidateSamFile.py", line 90, in parse_reports
>     data = _parse_reports_by_type(self)
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/site-packages/multiqc/modules/picard/ValidateSamFile.py", line 124, in _parse_reports_by_type
>     first_line = filehandle.readline().rstrip()
>   File "/N/u/parkanha/Carbonate/Application/MultiQC/conda-env/lib/python3.9/codecs.py", line 322, in decode
>     (result, consumed) = self._buffer_decode(data, self.errors, final)
> UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdb in position 9: invalid continuation byte
> ============================================================
> [INFO   ]        samtools : Found 1 rmdup reports
> [INFO   ]         multiqc : Compressing plot data
> [INFO   ]         multiqc : Report      : multiqc_report.html
> [INFO   ]         multiqc : Data        : multiqc_data
> [INFO   ]         multiqc : MultiQC complete


MultiQC successfully run for WGSCoverageMetrics.

**Comment by ewels** [comment]
Hi @parvathisudha,

This is the installation step:

```bash
conda install -c bioconda -c conda-forge multiqc
```

This is the command you use to run MultiQC once it is installed:

```bash
multiqc .
```

So the `multiqc .` command is not part of the installation - it's an example of what to do once it is installed.

The trailing `.` tells MultiQC to analyse files in your current directory (same as `./`). The full path that it's analysing is in the log you posted: `/geode2/home/u070/parkanha/Carbonate/Application/MultiQC`

So basically there is no problem with the installation I think. Just run MultiQC on some analysis files and not the installation directory 😄 As in, run `multiqc /path/to/my/analysis`

Based on your final comment:

> MultiQC successfully run for WGSCoverageMetrics.

I guess that everything is now working for you with a different run on some data? (🤞🏻) So I will close the issue. Let me know if any of the above is unclear or if you're having more problems.

Phil
